src/schema/connections.ts

```diff
@@ -18,7 +18,7 @@ import { getObjectsFromUrls } from './apiHelper';
 
 import { GraphQLInt, GraphQLList } from 'graphql';
 
-import type { GraphQLObjectType, GraphQLFieldConfig } from 'graphql';
+import { GraphQLObjectType, GraphQLFieldConfig } from 'graphql';
```

Comment:
Please keep `import type` and please enable `isolatedModules` in `tsconfig.json`